src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ProgressBar } from '../ProgressBar';
+import { OverlayTrigger } from '../OverlayTrigger';
+import { Tooltip } from '../Tooltip';
+import { labelClasses, mainDivClasses, barStyle } from './helpers';
+
+const randomId = () => Date.now();
+
+const AvailableTooltipFunction = (max, now) => (
+  <Tooltip id={randomId()}>Available {max - now} %</Tooltip>
+);
+
+const UsedTooltipFunction = now => (
+  <Tooltip id={randomId()}>Used {now} %</Tooltip>
+);
+
+const UtilizationBar = ({
+  min,
+  max,
+  now,
+  availableTooltipFunction,
+  usedTooltipFunction,
+  descriptionPlacementTop,
+  description,
+  thresholdWarning,
+  thresholdError,
+  label,
+  className
+}) => (
+  <div
+    className={mainDivClasses(
+      !descriptionPlacementTop,
+      className,
+      description,
+      label
+    )}
+  >
+    <div className={descriptionPlacementTop ? null : 'progress-bar'}>
+      {label && (
+        <span className={labelClasses(descriptionPlacementTop)}>
+          <strong className="label-strong label-text">
```

Comment:
Looks like these classes should not be added when not `descriptionPlacementTop`. Removing them matches https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/progress-bars.html